packages/braintree-integration/src/braintree-paypal/braintree-paypal-customer-strategy.ts

```diff
@@ -62,12 +62,17 @@ export default class BraintreePaypalCustomerStrategy implements CustomerStrategy
             );
         }
 
-        await this.paymentIntegrationService.loadPaymentMethod(methodId);
+        let state = this.paymentIntegrationService.getState();
+        let paymentMethod: PaymentMethod<BraintreeInitializationData>;
+
+        try {
+            paymentMethod = state.getPaymentMethodOrThrow(methodId);
+        } catch (_e) {
+            state = await this.paymentIntegrationService.loadPaymentMethod(methodId);
```

Comment:
It seems like we'll send another request for the same method if `getPaymentMethodOrThrow(methodId)` is executed in the middle of the first try.
Perhaps we can avoid this by only initializing a method after `loadPaymentMethodByIds` is done in `checkout-js`. `loadPaymentMethod(methodId)` should read from checkout store whenever possible.

Reply:
As a result, we don't need to change the existing payment methods if we make `loadPaymentMethod` do nothing when a method is already loaded.